data/json/npcs/hells_raiders/TALK_BANDIT.json

```diff
@@ -0,0 +1,112 @@
+[
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "npc_has_var": "first_meeting",
+      "type": "dialogue",
+      "context": "first_meeting",
+      "value": "yes",
+      "no": "Nice to see a new face around here, you one of us?  Welcome to the club, try not to step on anyone's toes.",
+      "yes": "<neutral_greet>"
+    },
+    "speaker_effect": { "effect": { "npc_add_var": "first_meeting", "type": "dialogue", "context": "first_meeting", "value": "yes" } },
+    "responses": [
+      {
+        "text": "<neutral_greet>",
+        "topic": "TALK_HELLS_RAIDERS_BANDIT_2",
+        "condition": { "npc_has_var": "first_meeting", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      },
+      {
+        "text": "Nice to meet you.",
+        "topic": "TALK_HELLS_RAIDERS_BANDIT_2",
+        "condition": { "not": { "npc_has_var": "first_meeting", "type": "dialogue", "context": "first_meeting", "value": "yes" } }
+      },
+      {
+        "text": "Nice to meet you.  Can't stick around though.",
+        "topic": "TALK_DONE",
+        "condition": { "not": { "npc_has_var": "first_meeting", "type": "dialogue", "context": "first_meeting", "value": "yes" } }
+      },
+      {
+        "text": "Hate to run, but I gotta go.  See ya.",
+        "topic": "TALK_DONE",
+        "condition": { "npc_has_var": "first_meeting", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_2",
+    "type": "talk_topic",
+    "dynamic_line": [ "What'cha want?", "Make it quick.", "I've got <name_b>s to beat up, not much time to talk." ],
+    "responses": [
+      { "text": "Anything that you need done around here?", "topic": "TALK_MISSION_LIST" },
+      { "text": "Who exactly are you guys anyway?", "topic": "TALK_HELLS_RAIDERS_BANDIT_WHO" },
+      { "text": "What do you do these days?", "topic": "TALK_HELLS_RAIDERS_BANDIT_JOB" },
+      { "text": "Could you loan a poor soul any supplies?", "topic": "TALK_HELLS_RAIDERS_BANDIT_NOBORROW" },
+      { "text": "Would you like to buy anything?", "topic": "TALK_HELLS_RAIDERS_BANDIT_NOTRADE" },
+      { "text": "Goodbye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_HELLS_RAIDERS_BANDIT_WHO",
+    "type": "talk_topic",
+    "dynamic_line": "Some people call us thieves, others scum, the <swear> government calls us the Hell's Raiders.  We're just some folks doin' what we need to survive is all.",
```

Comment:
They consider themselves the legitimate government.

Reply:
The [Faction Design Doc](https://cataclysmdda.org/lore-factions.html#hells-raiders) lists them all as criminal gangs with semi-loose power structures. 
Just out of curiosity, where wa this new vision of them lore-wise hashed out?

Reply:
I had thought it was always somewhat explicit that the leaders of hell's raiders had been responsible for the rhode island secession, but I guess it was not.
Erk talked about this in the April 8th conversation in lore and design.

Reply:
I'm not quite sure what to do from that. Nonetheless, this leaves inconsistencies - the currency doesn't fit if they aren't, and the dialogue doesn't fit they are.

Reply:
For the time being, I'll change the currency around and leave the dialogue as is, and see what happens.